pkg/frontend/validate.go

```diff
@@ -78,7 +78,33 @@ func (f *frontend) validateOpenShiftUniqueKey(ctx context.Context, doc *api.Open
 // prevent mischief
 var rxKubernetesString = regexp.MustCompile(`(?i)^[-a-z0-9.]{0,255}$`)
 
+func validatePermittedClusterwideObjects(groupKind string) bool {
+	validApiGroups := []string{
+		"nodes",
+		"aro.openshift.io",
+		"config.openshift.io",
+		"console.openshift.io",
+		"machine.openshift.io",
+		"machineconfiguration.openshift.io",
+		"imageregistry.operator.openshift.io",
+		"certificatesigningrequest.certificates.k8s.io",
+		"apirequestcounts.apirequestcount.openshift.io",
+		"rbac.authorization.k8s.io",
+		"authorization.openshift.io",
```

Comment:
Any chance we could put `operator.openshift.io` in here?  

`dnses, imagecontentsourcepolicies, kubeapiserver, schedulers, openshiftapiservers` are all content we've used in the past. 

`security.openshift.io` -> securitycontextconstraints are also useful, but might expose customer SCCs.  But they are just names so they might be fine.

Reply:
if this was a `map[string]bool` that'd reduce the complexity a little bit

Reply:
+1 on `operator.openshift.io`, since that includes things such as network/DNS/ingress which are essential for us for determining a cluster state.

Reply:
the contents of an SCC can contain usernames (unfortunately)

but I'm cool with operator.openshift.io